inst/extdata/load_evaporation.R

```diff
@@ -98,24 +97,24 @@ evaporation_stats <- function(evaporation_matrices,
                               geo_mask)
 {
   pot_evap_stat <- file_info
-  
+
 
   for (i in 1:length(evaporation_matrices)) {
-    
-    #keep only Berlin grid cells  
-    Berlin_values <- evaporation_matrices[[i]] * geo_mask
-    
+
+    #keep only Berlin grid cells
+    berlin_values <- evaporation_matrices[[i]] * geo_mask
+
     #correct unit to mm
-    Berlin_values <- Berlin_values / 10
-    
-    pot_evap_stat$mean[i] <- mean(Berlin_values, na.rm = TRUE)
-    pot_evap_stat$sd[i] <- sd(Berlin_values, na.rm = TRUE)
-    pot_evap_stat$min[i] <- min(Berlin_values, na.rm = TRUE)
-    pot_evap_stat$max[i] <- max(Berlin_values, na.rm = TRUE)
-    
+    berlin_values <- berlin_values / 10
+
+    pot_evap_stat$mean[i] <- mean(berlin_values, na.rm = TRUE)
+    pot_evap_stat$sd[i] <- sd(berlin_values, na.rm = TRUE)
+    pot_evap_stat$min[i] <- min(berlin_values, na.rm = TRUE)
+    pot_evap_stat$max[i] <- max(berlin_values, na.rm = TRUE)
```

Comment:
You avoid code repetition by using a one-line helper function `get_stats()`:
```suggestion
    get_stats <- function(fun) fun(berlin_values, na.rm = TRUE)
    pot_evap_stat$mean[i] <- get_stats(mean)
    pot_evap_stat$sd[i] <- get_stats(sd)
    pot_evap_stat$min[i] <- get_stats(min)
    pot_evap_stat$max[i] <- get_stats(max)
```
Having said that I see that you can further "simplify". I did not test it but you may want to try this out:
```suggestion
    pot_evap_stat <- cbind(pot_evap_stat, lapply(
      list(mean = mean, sd = sd, min = min, max = max), 
      function(fun) fun(berlin_values, na.rm = TRUE)
    ))
```